### ISSUE TITLE
File not found exception when loading mp3

### ISSUE THREAD

**Issue by jeruliu** [jiaaro/pydub#254]
### Steps to reproduce
sound = AudioSegment.from_mp3("C:/dev/butterfly.mp3")

### Expected behavior
Tell us what should happen

### Actual behavior
gettting "FileNotFoundError: [WinError 2]" from the console, do you support windows system?

### Your System configuration
- Python version: 3.6
- Pydub version: latest
- ffmpeg or avlib?: 
- ffmpeg/avlib version: 

### Is there an audio file you can include to help us reproduce?
You can include the audio file in this issue - just put it in a zip file and drag/drop the zip file into the github issue.

**Comment by jiaaro** [comment]
Yes pydub works on windows - usually this error happens either because ffmpeg can’t be located or because the file you’re trying to convert doesn’t exist (I.e. the path is wrong)
